services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeBuilder.kt

```diff
@@ -59,18 +64,8 @@ data class HankeBuilder(
      */
     fun save(): Hanke {
         val createdHanke = create()
-        return hankeService.updateHanke(
-            hanke.copy(id = createdHanke.id, hankeTunnus = createdHanke.hankeTunnus).apply {
-                omistajat = hanke.omistajat.onEach { it.id = null }
-                rakennuttajat = hanke.rakennuttajat.onEach { it.id = null }
-                toteuttajat = hanke.toteuttajat.onEach { it.id = null }
-                muut = hanke.muut.onEach { it.id = null }
-                tyomaaKatuosoite = hanke.tyomaaKatuosoite
-                tyomaaTyyppi = hanke.tyomaaTyyppi
-                alueet = hanke.alueet
-                tormaystarkasteluTulos = hanke.tormaystarkasteluTulos
-            }
-        )
+        hanke.copy(id = createdHanke.id, hankeTunnus = createdHanke.hankeTunnus).apply {}
+        return hankeService.updateHanke(createdHanke.hankeTunnus, hanke.toModifyRequest { null })
```

Comment:
Just `hanke.toModifyRequest()`?

Reply:
This is different from just `hanke.toModifyRequest()`. This overrides the idMapper function with one that always returns null, while the default returns the unchanged id. I used a named parameter here to maybe make it clearer.